DigitalLearningSolutions.Data.Migrations/202203161448_RestructureAdminAndDelegateTables.cs

```diff
@@ -0,0 +1,195 @@
+            Rename.Column("Forename").OnTable("AdminAccounts").To("Forename_deprecated");
+            Rename.Column("Surname").OnTable("AdminAccounts").To("Surname_deprecated");
+            Rename.Column("Approved").OnTable("AdminAccounts").To("Approved_deprecated");
+            Rename.Column("PasswordReminder").OnTable("AdminAccounts").To("PasswordReminder_deprecated");
+            Rename.Column("PasswordReminderHash").OnTable("AdminAccounts").To("PasswordReminderHash_deprecated");
+            Rename.Column("PasswordReminderDate").OnTable("AdminAccounts").To("PasswordReminderDate_deprecated");
+            Rename.Column("EITSProfile").OnTable("AdminAccounts").To("EITSProfile_deprecated");
+            Rename.Column("TCAgreed").OnTable("AdminAccounts").To("TCAgreed_deprecated");
+            Rename.Column("FailedLoginCount").OnTable("AdminAccounts").To("FailedLoginCount_deprecated");
+            Rename.Column("ProfileImage").OnTable("AdminAccounts").To("ProfileImage_deprecated");
+            Rename.Column("SkypeHandle").OnTable("AdminAccounts").To("SkypeHandle_deprecated");
+            Rename.Column("PublicSkypeLink").OnTable("AdminAccounts").To("PublicSkypeLink_deprecated");
+            Rename.Column("ResetPasswordID").OnTable("AdminAccounts").To("ResetPasswordID_deprecated");
+
+            Rename.Table("Candidates").To("DelegateAccounts");
+            Alter.Table("DelegateAccounts").AddColumn("UserID").AsInt32().Nullable().ForeignKey("Users", "ID");
+            Alter.Column("DateRegistered").OnTable("DelegateAccounts").AsDateTime().Nullable();
+
+            Delete.Index("IX_Candidates_CandidateNumber").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_LastName").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_FirstName_LastName").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_Active_CentreID").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_DateRegistered").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_FirstName").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID").OnTable("DelegateAccounts");
+            Alter.Column("CandidateNumber").OnTable("DelegateAccounts").AsString(20).NotNullable();
+            Create.Index("IX_Candidates_CandidateNumber").OnTable("DelegateAccounts").OnColumn("CandidateNumber")
+                .Ascending().WithOptions().Unique().WithOptions().NonClustered();
+            Create.Index("IX_Candidates_CentreID_FirstName_LastName").OnTable("DelegateAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("FirstName").Ascending().OnColumn("LastName").Ascending().WithOptions().NonClustered();
+            Create.Index("IX_Candidates_CentreID_LastName").OnTable("DelegateAccounts").OnColumn("CentreID").Ascending()
+                .OnColumn("LastName").Ascending().WithOptions().NonClustered();

```

Comment:
These indexes will reference deprecated columns, do we want this?

Reply:
Steve has confirmed that we don't need to add back any indexes based on deprecated columns